alf/utils/math_ops.py

```diff
@@ -457,3 +458,36 @@ def transform(self, x):
 
     def inverse_transform(self, y):
         return y.sign() * ((y / self._alpha).abs().exp() - 1)
+
+
+@alf.configurable
```

Comment:
These hindsight related functions shouldn't be put in a general file like `math_ops.py`.

Reply:
Both data_transformer and suite_robotics use these functions, so util is perhaps a reasonable place?  Should I create a utils/reward_functions.py?

Reply:
> Both data_transformer and suite_robotics use these functions, so util is perhaps a reasonable place? Should I create a utils/reward_functions.py?

These calculations are very simple (basically one line) and don't have to be factored into functions.